configs/mesh_thread.json

```diff
@@ -32,7 +33,15 @@
         "NUCLEO_F401RE": {
             "LED": "LED_RED",
             "BUTTON": "USER_BUTTON"
+        },
+        "KW24D": {
+            "mbed-mesh-api.heap-size": 12000,
```

Comment:
12k can be good for static configuration but commissioning doesn't work well below 14-15k heap